<!--
  2 failure:  :microscope: Ther..., :disappointed_rel...
  1 warning:  :mag: Can't find ...
  1 messages
  
  DangerID: danger-id-default;
-->

<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>

  :microscope: There are library changes, but no changes to the unit tests. That's OK as long as you're refactoring existing code, but will require an admin to merge this PR. Please see <a href="https://github.com/janvennemann/titanium_mobile/blob/TIMOB-26668/README.md#unit-tests">README.md#unit-tests</a> for docs on unit testing.
  </td>
    </tr>
  
<tr>
      <td>:no_entry_sign:</td>
      <td>

  :disappointed_relieved: `npm test` failed. See below for details.
  </td>
    </tr>
  </tbody>
</table>


<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  :mag: Can't find junit reports at `./junit.*.xml`, skipping generating JUnit Report.
  </td>
    </tr>
  </tbody>
</table>


<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Messages</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:book:</td>
      <td>

  ```
> titanium-mobile@8.0.0 test /Users/build/jenkins/workspace/ium-sdk_titanium_mobile_PR-10550
> grunt

Running "appcJs:src:lintOnly" (appcJs) task

Running "eslint:src" (eslint) task

Running "checkFormat:ios" (checkFormat) task
Fatal error: Error: Formatting incorrect on "iphone/Classes/TiUIScrollView.m", proposed changes: <?xml version='1.0'?>
<replacements xml:space='preserve' incomplete_format='false'>
<replacement offset='10503' length='3'>&#10;</replacement>
<replacement offset='11016' length='3'>&#10;</replacement>
</replacements>

npm ERR! Test failed.  See above for more details.

```
  </td>
    </tr>
  </tbody>
</table>


<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

--- a/iphone/Classes/TiUIScrollView.m
+++ b/iphone/Classes/TiUIScrollView.m
@@ -316,7 +316,7 @@
 - (void)scrollToBottom
 {
   /*
-     * Calculate the bottom height & width and, sets the offset from the 
+     * Calculate the bottom height & width and, sets the offset from the
      * content view’s origin that corresponds to the receiver’s origin.
      */
   UIScrollView *currScrollView = [self scrollView];
@@ -362,6 +362,7 @@
 
 - (void)setRefreshControl_:(id)args
 {
+#ifdef USE_TI_UIREFRESHCONTROL
   ENSURE_SINGLE_ARG_OR_NIL(args, TiUIRefreshControlProxy);
   [[refreshControl control] removeFromSuperview];
   RELEASE_TO_NIL(refreshControl);
@@ -374,6 +375,7 @@
       [[self scrollView] addSubview:refreshControl.control];
     }
   }
+#endif
 }
 
 - (void)setShowHorizontalScrollIndicator_:(id)value

